Dockerfile

```diff
@@ -17,6 +17,7 @@ COPY --from=builder /usr/src/app/node_modules /usr/src/app/node_modules
 COPY package.json /usr/src/app
 COPY index.js /usr/src/app
 RUN mkdir ~/.ssh
-RUN ssh-keyscan -H wetty-ssh >> ~/.ssh/known_hosts
+RUN echo 'ssh-keyscan -H ${SSHHOST} >> ~/.ssh/known_hosts' > start_script.sh
+RUN echo 'yarn start' >> start_script.sh
+RUN chmod +x start_script.sh
 
-ENTRYPOINT [ "node", "." ]
```

Comment:
removing entry point will break the container for anyone currently using it

Reply:
I can change it to sh start_script.sh if the previous thing is good with you.